index.html

```diff
@@ -10,7 +10,145 @@
+        <rect width="26.4618" height="4.7339" transform="matrix(0.757375 0.65298 0.65298 -0.757375 0.18457 3.58533)"
+            fill="#6070FF" />
+    </svg>
+    <svg class="circle-top-middle" width="11" height="12" viewBox="0 0 11 12" fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <ellipse cx="5.30398" cy="6.21088" rx="5.5" ry="5" transform="rotate(52.9708 5.30398 6.21088)" fill="#CBCDFF" />
+    </svg>
+    <!-- Top Middle Shapes Ends Here -->
+
+    <!-- Bottom Left Shapes Starts Here -->
+    <svg class="bottom-left-circle" width="15" height="15" viewBox="0 0 15 15" fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <circle cx="7.5" cy="7.49988" r="7" fill="white" stroke="#CBCDFF" />
+    </svg>
+    <!-- Bottom Left Shapes Ends Here -->
+
+    <!-- Bottom Middle Shapes Starts Here -->
+    <svg class="bottom-middle-rectangle" width="68" height="39" viewBox="0 0 68 39" fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <rect x="4.82422" y="-0.00012207" width="68.0631" height="12.1762" transform="rotate(23.34 4.82422 -0.00012207)"
+            fill="#EBEBFF" />
+    </svg>
+    <svg class="bottom-middle-circle" width="20" height="21" viewBox="0 0 20 21" fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <circle cx="10" cy="10.0956" r="10" fill="#6070FF" />
+    </svg>
+
+    <!-- Bottom Middle Shapes Ends Here -->
+
+
+    <!-- Bottom Right Shapes Starts Here -->
+    <svg class="bottom-right-above-circle" width="25" height="13" viewBox="0 0 25 13" fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <path
+            d="M0.496962 -0.000453282C0.278168 3.19352 1.33714 6.3436 3.44091 8.75679C5.54468 11.17 8.52093 12.6486 11.7149 12.8674C14.9089 13.0862 18.059 12.0272 20.4722 9.92347C22.8853 7.8197 24.364 4.84345 24.5828 1.64948L0.496962 -0.000453282Z"
+            fill="#6070FF" />
+    </svg>
+    <svg class="bottom-right-below-circle" width="31" height="26" viewBox="0 0 31 26" fill="none"
+        xmlns="http://www.w3.org/2000/svg">
+        <path
+            d="M26.9067 25.4993C28.4263 23.6894 29.4982 21.5472 30.0358 19.246C30.5734 16.9448 30.5616 14.5493 30.0012 12.2535C29.4408 9.95778 28.3477 7.82624 26.8103 6.03152C25.2729 4.2368 23.3345 2.82935 21.152 1.92312C18.9695 1.01689 16.6043 0.637356 14.2478 0.815252C11.8914 0.993148 9.60991 1.72347 7.58819 2.94709C5.56648 4.17071 3.86134 5.85322 2.61082 7.85841C1.3603 9.86359 0.599568 12.1351 0.390208 14.489L6.84065 15.0627C6.95992 13.7217 7.39331 12.4276 8.10573 11.2853C8.81815 10.1429 9.78957 9.18438 10.9413 8.48729C12.0931 7.79019 13.3928 7.37413 14.7353 7.27278C16.0778 7.17144 17.4253 7.38765 18.6686 7.90393C19.912 8.42021 21.0163 9.22204 21.8922 10.2445C22.768 11.2669 23.3908 12.4813 23.71 13.7892C24.0292 15.0971 24.036 16.4618 23.7297 17.7727C23.4235 19.0837 22.8128 20.3042 21.9471 21.3353L26.9067 25.4993Z"
+            fill="#EBEBFF" />
+    </svg>
+
+    <!-- Bottom Right Shapes Ends Here -->
+
+
```

Comment:
- [ ] As the last code reviewer said:

> For the background image, kindly use background-image in CSS instead of HTML elements, since this image doesn't contribute to the content of the page it's just for styling. Hint: you can download the whole background image from Figma instead of taking the shapes one by one.

- [ ] Please 🙏🏽 complete all the previous requirements before asking for a new code review request, because you have limited chances to approve the project.